WordPress/src/main/res/layout/fragment_my_site.xml

```diff
@@ -0,0 +1,309 @@
+<ScrollView xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    android:layout_width="fill_parent"
```

Comment:
Minor nit: use `match_parent` rather than `fill_parent` everywhere. It actually makes no difference to the layout (they do the exact same thing), but `match_parent` is the "newer" version of the two. [Details](http://java.dzone.com/articles/android-tips-matchparent-vs).